.travis.yml

```diff
@@ -30,9 +30,9 @@ after_success:
 
 deploy:
   provider: pypi
-  user: "__token__"
+  user: edx
```

Comment:
@feanil: 
* Our docs say to use `"__token__"`: https://openedx.atlassian.net/wiki/spaces/EdxOps/pages/157464369/How+to+connect+a+repository+to+Travis+CI
* Our repos seem to use `edx`: https://github.com/edx/edx-rest-api-client/blob/master/.travis.yml#L31
Do you know why the difference?

Reply:
This was a change in PyPI, they now support and encourage the use of API tokens for uploads so the CI system doesn't need the master password for your PyPI account: https://pyfound.blogspot.com/2020/01/start-using-2fa-and-api-tokens-on-pypi.html

Reply:
Ok.  I'll back out this change and assume I just screwed up the encryption the first time.